Source/Diagnostics/ReducedDiags/FieldMomentum.cpp

```diff
@@ -0,0 +1,147 @@
+/* Copyright 2019-2020
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "FieldMomentum.H"
+#include "WarpX.H"
+#include "Utils/WarpXConst.H"
+#include "Utils/CoarsenIO.H"
+
+#include <AMReX_REAL.H>
+
+#include <iostream>
+#include <cmath>
```

Comment:
`iostream` is a really big header, I think we only need `ostream` here for `std::ofstream` & `std::endl`:
```suggestion
#include <cmath>
#include <ostream>
#include <string>
```
```